server/src/test/kotlin/tech/figure/objectstore/gateway/server/ObjectStoreGatewayServerTest.kt

```diff
@@ -60,9 +65,9 @@ class ObjectStoreGatewayServerTest {
 
     lateinit var server: ObjectStoreGatewayServer
 
-    val defaultGranter: String = "accessGranter"
-    val scopeAddress = "scopeAddress"
-    val defaultGrantee = "grantee"
+    val defaultGranter: String = genRandomAccount().bech32Address
```

Comment:
Had to tweak all these tests to ensure they pass the bech32 validation.  I could've done a workaround that allowed verification to be bypassed, but I kinda like the fact that we now have like a bunch of tests coming at this from a ton of different unexpected angles